Payjoin.md

```diff
@@ -0,0 +1,42 @@
+# BTCPay Server Payjoin Guide
+
+This document explains how to use BTCPay Server's Payjoin feature. For a detailed, technical explanation of how payjoin is implemented, check the [Payjoin Specififcations](Payjoin-spec.md)
+
+## Enabling Payjoin as a merchant
+
+1. Create a store
+2. Configure a hot wallet for your derivation scheme
+   * Store => Derivation Scheme => BTC => Modify => Import from => a new/existing seed => Is hot wallet checked and using either segwit or segwit wrapped as an address type => Generate.
+   * Don't forget to click Confirm!
+3. Enable Payjoin/P2EP in the "General Settings" and click "Save"
+
+It's important to note that you will need at least 1 UTXO for payjoin to work.
```

Comment:
@Kukks  I would add these, it may seem like too much, but people are unfamiliar with these functions so I guess it;s good to explains steps visually after describing them.
![payjoin a newseed](https://user-images.githubusercontent.com/36959754/79072363-f3060d00-7ce0-11ea-9093-f67bdbfb776c.png)

![payjoin a is hot wallet](https://user-images.githubusercontent.com/36959754/79072367-f8635780-7ce0-11ea-9e13-deafb3bd1be3.png)

![enable payjoin](https://user-images.githubusercontent.com/36959754/79072348-e7b2e180-7ce0-11ea-8546-ef24515c7d67.PNG)